airflow/config_templates/config.yml

```diff
@@ -1251,6 +1251,13 @@ metrics:
       type: string
       example: ~
       default: "False"
+    otel_service:
+      description: |
+        The default service name of traces.
+      version_added: 2.10.1
```

Comment:
needs to update to 2.10.3 and confirm with @ephraimbuddy that this PR is for bugfix release otherwise it must be 2.11

Reply:
changed for 2.10.3